readme.md

```diff
@@ -554,6 +562,28 @@ is.all(is.string, '🦄', [], 'unicorns');
 //=> false
 ```
 
+##### .validLength(value)
+
+Returns `true` if the value is a safe integer that is greater than or equal to zero.
+
+This can be useful to confirm that a value is a valid count of something, ie. 0 or more.
+
+##### .whitespaceString(value)
+
+Returns `true` if the value is a string with only whitespace characters.
+
+## Named Exports
+
+Named exports allow tooling to perform tree-shaking, potentially reducing bundle size by including only code from the methods that are used.
+
+Every method listed above is available as a named export. Each method is prefixed by either `is` or `assert` depending on usage.
+
+For example:
+
+```js
+import {assertNull, isUndefined} from '@sindresorhus/is';
+```
```

Comment:
I think this should go at the end of the `Usage` section

Reply:
Good idea, I moved it up underneath the Usage section.